backend/substrapp/serializers/ledger/objective/serializer.py

```diff
@@ -24,6 +26,7 @@ def create(self, validated_data):
         permissions = validated_data.get('permissions')
         test_data_manager_key = validated_data.get('test_data_manager_key', '')
         test_data_sample_keys = validated_data.get('test_data_sample_keys', [])
+        metadata = validated_data.get('metadata')
```

Comment:
You use `metadata = validated_data.get('metadata', {})` everywhere else, which one should it be?

Reply:
The `{}` wasn't necessary, I removed it.